src/content/fr/2019/fonts.md

```diff
@@ -0,0 +1,682 @@
+        <td class="numeric">0.1%</td>
+      </tr>
+      <tr>
+        <td>fonts.jimstatic.com</td>
+        <td class="numeric">0.1%</td>
+        <td class="numeric">0.2%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Top 20 des hébergements de polices de caractère par requêtes.") }}</figcaption>
+</figure>
+
+La domination de Google Fonts ici était à la fois surprenante et non surprenante. Elle n'était pas surprenante dans la mesure où je m'attendais à ce que le service soit le plus populaire et surprenant par la simple domination de sa popularité. 75 % des demandes de polices c'est étonnant. TypeKit était en deuxième position,avec un pourcentage à un chiffre, la bibliothèque Bootstrap occupant une troisième place encore plus éloignée.
+
+{{ figure_markup(
+  caption="Pourcentage des pages qui incluent un lien vers des feuilles de style Google Fonts dans l'entête (<code><head></code>) du document.",
+  content="29%",
+  classes="big-number"
+)
+}}
+
+Si l'utilisation élevée des Google Fonts ici est très impressionnante, il est également à noter que seulement 29% des pages comprenaient un élément Google Fonts `<link>`. Cela pourrait signifier plusieurs choses :
+
+- Lorsque des pages utilisent des Google Fonts, elles utilisent _beaucoup_ de Google Fonts. Elles sont fournies sans coût monétaire, après tout. Peut-être sont-elles utilisées dans un éditeur WYSIWYG populaire ? Cela semble être une explication très probable.
+- Ou il se pourrait de façon plus improbable que beaucoup de gens utilisent les Google Fonts avec `@import` au lieu de `<link>`.
+- Ou, si nous voulons aller plus loin dans des scénarios très improbables, cela pourrait signifier que beaucoup de gens utilisent des polices Google avec un [HTTP `Link:` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link) à la place.
+
+{{ figure_markup(
+  caption="Pourcentage des pages qui incluent un lien vers des feuilles de style Google Fonts comme premier noeud dans l'entête du document (<code><head></code>).",
+  content="0.4%",
+  classes="big-number"
+)
+}}
+
+La documentation de Google Fonts encourage que le `<link>` pour le CSS Google Fonts soit placé comme premier enfant dans le `<head>` d'une page. C'est une demande forte ! Dans la pratique, ce n'est pas courant car seulement un demi pour cent de toutes les pages (environ 20 000 pages) ont suivi ce conseil.
+
+De plus, si une page utilise les éléments `preconnect` ou `dns-prefetch` comme `<link>`, ceux-ci passeront de toute façon avant les polices CSS de Google. Lisez la suite pour en savoir plus sur des indications de ressources.
+
+### Accélérer l'hébergement tierce-partie
+
+Comme mentionné ci-dessus, un moyen très simple d'accélérer les demandes de polices Web sur un hébergement tierce-partie est d'utiliser la fonction `preconnect` [resource hint](./resource-hints).
+
+{{ figure_markup(
+  caption="Pourcentage des pages mobiles qui se préconnectent à un serveur de web font.",
```

Comment:
`un serveur de web font`/`un serveur de polices web`

Reply:
Updated in https://github.com/HTTPArchive/almanac.httparchive.org/commit/6df484a83f05c285020d15faa5a0207a42c519e7